var x = 1;

try do
    var x = 2
catch ObjectException do
    print "ObjectException 1";
catch SyntaxException do
    print "SyntaxException 2";
end_try

print x;
